### PR TITLE
French Subtitles & Game Text

### PR DESCRIPTION
**Pull request by 0xMisagi**
## Cutscenes
- [x] intro
- [x] sidekick
- [x] oracle
- [x] village1
- [x] beach
- [x] jungle
- [x] misty
- [x] firecanyon
- [x] village2
- [x] swamp
- [x] village3
- [x] lavatube
- [x] citadel
- [x] finalboss

## Hints
- [x] sidekick
- [x] training
- [x] village1
- [x] beach
- [x] jungle
- [x] misty
- [x] firecanyon
- [x] village2
- [x] swamp
- [x] rolling
- [x] sunken
- [x] ogre
- [x] village3
- [x] snowy
- [x] spidercave
- [x] lavatube
- [x] citadel
- [x] finalboss
- [x] title

## Text
- [x] game_text_fr.gs

### CODE REVIEW

File: game/assets/jak1/text/game_text_en.gs [hunk 1]
```diff
@@ -355,6 +355,8 @@
         "DANSK")
 (#x1116 "NORSK"
         "NORSK")
+(#x115 "FRENCH"
+       "FRENCH")
```
Comment (ManDude) [inline game/assets/jak1/text/game_text_en.gs:359]: Remove this.

File: game/assets/jak1/text/game_text_fr.gs [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+(#x1107 "VARIANTE 2")
+
+(#x1110 "ENGLISH (UK)")
+(#x1111 "PORTUGUÊS")
+(#x1112 "PORTUGUÊS (BRASIL)")
+(#x1113 "SUOMI")
+(#x1114 "SVENSKA")
+(#x1115 "DANSK")
+(#x1116 "NORSK")
+(#x115 "FRANÇAIS")
```
Comment (ManDude) [inline game/assets/jak1/text/game_text_fr.gs:195]: This line isn't necessary.

**Review by ManDude — CHANGES_REQUESTED** [review]

File: game/assets/jak1/text/game_text_en.gs (hunk 1 above)
Comment (0xMisagi) [reply game/assets/jak1/text/game_text_en.gs]: Done.

**Review by ManDude — APPROVED** [review]

**Comment by 0xMisagi** [comment]
Ignore the commit ([9ab8732])

**Comment by 0xMisagi** [comment]
Ready for offline tests. Let me know if there's any outliers (hopefully there shouldn't be)

subtitles are limited to 2 lines max on the default 4x3 resolution as told in the documentation

And game_text_fr.gs options menu entries are limited to a single line (manually checked)

**Review by ManDude — APPROVED** [review]

**Comment by xTVaser** [comment]
ok to test

**Pull request merged**